src/ros_filter.cpp

```diff
@@ -1957,6 +1974,42 @@ namespace RobotLocalization
   }
 
   template<typename T>
+  bool RosFilter<T>::enableFilterSrvCallback(std_srvs::Empty::Request& request,
+                                             std_srvs::Empty::Response&)
+  {
+    RF_DEBUG("\n[" << ros::this_node::getName() << ":]" << " ------ /RosFilter::enableFilterSrvCallback ------\n");
+    if (enabled_) {
+      ROS_WARN_STREAM("[" << ros::this_node::getName() << ":] Asking for enabling filter service, but the filter was already enabled! Use param disabled_at_startup.");
+    } else {
+      ROS_INFO_STREAM("[" << ros::this_node::getName() << ":] Enabling filter...");
+      enabled_ = true;
+    }
+    return true;
+  }
+
+  template<typename T>
+  std::string RosFilter<T>::tfFailureReasonString(const tf2_ros::FilterFailureReason reason)
```

Comment:
Thanks for implementing this method. Not sure how that got added to the header and never defined. However, as it's not used, I think I'm more inclined to remote the method from the header.